webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -72,9 +72,9 @@ export const combineMenuItems = data => {
 const createOrgItem = orgs => {
   const anyOrg = {
     name: ANY_ORGANIZATION_TEXT,
-    url: '/organizations/clear',
     onClick: () => {
       changeOrganization(ANY_ORGANIZATION_TEXT);
+      window.location.assign('/organizations/clear');
```

Comment:
should we use tfm.navigation.visit or react router history for this? Also, why is this different to the other menu items that just use url?

Reply:
Because other menu items are urls that lead to pages, `clear` and `select` are urls that preform an action and then do: `redirect_back_or_to root_url`.
When I click "any org" from the header the server redirects me to the page I'm on, when I click "any org" from the navigation it redirects me to `/org/clear` and creates a loop.
I don't want to use `visit` as  `window.location.href = url;` is a bit difficult to test if I remember correctly.
```
export const visit = url => {
  window.location.href = url;
};
```
I didn't have any luck getting it to work with react router history.